pandas/tseries/tests/test_timedeltas.py

```diff
@@ -1083,6 +1083,40 @@ def test_timedelta_hash_equality(self):
         ns_td = Timedelta(1, 'ns')
         self.assertNotEqual(hash(ns_td), hash(ns_td.to_pytimedelta()))
 
+    def test_min(self):
+        # left boundary is valid
+        min_td = pd.Timedelta(pd.Timedelta.min)
+        self.assertTrue(isinstance(min_td, pd.tslib.Timedelta))
```

Comment:
use ``self.assertIsInstance(min_td, pd.Timedelta)``

actually don't need to use ``pd.`` anywhere here